backend/src/main/java/gov/cdc/usds/simplereport/db/repository/PatientLinkRepository.java

```diff
@@ -9,11 +9,9 @@
 import org.springframework.data.jpa.repository.Query;
 
 public interface PatientLinkRepository extends EternalAuditedEntityRepository<PatientLink> {
-  List<PatientLink> findAllByTestOrderInternalIdIn(Collection<UUID> testOrderIds);
-
   @Query(
       value =
-          "select DISTINCT on (ordered.test_order_id) * from (select * from {h-schema}patient_link where test_order_id in :testOrderIds order by created_at desc ) ordered where test_order_id in :testOrderIds",
+          "select DISTINCT on (ordered.test_order_id) * from (select * from {h-schema}patient_link where test_order_id in :testOrderIds order by created_at desc ) ordered",
```

Comment:
removing extra where clause per @mehansen feedback, discussed multiple implementations and sub-querys tended to perform the best